src/ripple/protocol/Serializer.h

```diff
@@ -298,7 +297,7 @@ class Serializer
     }
 
     // low-level VL length encode/decode functions
-    static Blob encodeVL (int length);
+    static int encodeVL (int length, std::array<std::uint8_t, 4>& result);
```

Comment:
@scottschurr This code comes from a larger refactoring I was working on with serializer that's being sidelined for now. I'm keeping the branch around in case I ever get to return from it and those changes would go into that branch. I do agree with you, just not for this very tiny PR.